omnibus/config/software/supermarket-cookbooks.rb

```diff
@@ -41,6 +41,7 @@
     open("#{cookbooks_path}/solo.rb", "w") do |file|
       file.write <<-EOH.gsub(/^ {8}/, '')
         cookbook_path   "#{cookbooks_path}"
+        cache_path "/var/opt/supermarket/local-mode-cache"
```

Comment:
Looks like a `cache_path` of `/var/opt/supermarket` would work?

* local mode [would make](https://github.com/chef/chef/blob/master/chef-config/lib/chef-config/config.rb#L236) `/var/opt/supermarket/local-mode-cache`
* `file_cache_path` [would make](https://github.com/chef/chef/blob/master/chef-config/lib/chef-config/config.rb#L263) `/var/opt/supermarket/cache`